README.md

```diff
@@ -1,70 +1,65 @@
-# Getting Started with Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-
-## Available Scripts
-
-In the project directory, you can run:
-
-### `npm start`
-
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
-
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+# COVID METRICS WEBAPP
+- In this repo I was to select an API that provides numeric data about a topic that I would like and then build the webapp around it. The webapp will have several pages:
+  - one page with a list of items that could be filtered by some parameters.
+  - one page for the item details.
 
-### `npm test`
+## Project Description
+> Click [here](https://www.loom.com/share/6ce1cef478734b91a5feabdfaf41da22) to watch a video that gives a brief description of the project
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Demo
+> Click [here](tender-wescoff-adf161.netlify.app/) to see the live demo
```

Comment:
- [ ] While the live link in your PR works, I get a 404 error when I try to view the live link in your README. Kindly fix it.